run_analysis.sh

```diff
@@ -41,16 +51,23 @@ EMBEDDING=1                 # options: 0, 1
 
 # Build workspace
 STXS_FIT="inclusive"        # options: stxs_stage0, stxs_stage1p1, inclusive
-./datacards/produce_workspace.sh $ERA $STXS_FIT | tee ${ERA}_produce_workspace_${STXS_FIT}.log
+./datacards/produce_workspace.sh $ERA $STXS_FIT $METHOD | tee ${ERA}_produce_workspace_${STXS_FIT}.log
 
 # Run statistical inference
 #./combine/significance.sh $ERA | tee ${ERA}_significance.log
-./combine/signal_strength.sh $ERA $STXS_FIT | tee ${ERA}_signal_strength_${STXS_FIT}.log
-./combine/signal_strength.sh $ERA "robustHesse" | tee ${ERA}_signal_strength_robustHesse.log
-./combine/diff_nuisances.sh $ERA
+./combine/signal_strength.sh $ERA $STXS_FIT $DATACARDDIR/cmb/125 cmb ${METHOD}
+# ./combine/signal_strength.sh $ERA "robustHesse" $DATACARDDIR/cmb/125 cmb ${METHOD}
+# ./combine/diff_nuisances.sh $ERA
 #./combine/nuisance_impacts.sh $ERA
 
 # Make prefit and postfit shapes
-./combine/prefit_postfit_shapes.sh $ERA
-./plotting/plot_shapes.sh $ERA $STXS_SIGNALS $CATEGORIES $JETFAKES $EMBEDDING $CHANNELS
+
+# ./combine/prefit_postfit_shapes.sh ${ERA} ${STXS_FIT} ${DATACARDDIR}/cmb/125 ${METHOD}
+
+# PREFITFILE="${DATACARDDIR}/prefitshape-${era}-${METHOD}-${STXS_FIT}.root"
+# PLOTDIR=output/plots/${era}-${METHOD}_prefit-plots
+# [ -d $PLOTDIR ] || mkdir -p $PLOTDIR
+
+# ./plotting/plot_shapes.py -i $PREFITFILE -o $PLOTDIR -c ${CHANNELS[@]} -e $era $OPTION --categories $CATEGORIES --fake-factor --embedding --normalize-by-bin-width -l --train-ff $TRAINFF --train-emb $TRAINEMB
```

Comment:
Capitalize $era, $TRAINFF and $TRAINEMB are not set

Reply:
Either 
source utils/setup_cvmfs_sft.sh
source utils/setup_python.sh
will have to be executed before, or do the plotting in a seperate shell script again

Reply:
Also the png versions should still be produced